src/components/Game/Game.js

```diff
@@ -96,14 +138,18 @@ class GameView extends Component {
     };
 
     submit = () => {
-        const { lettersInSlots } = this.state;
+        const { letters, lettersInSlots } = this.state;
         const { dispatch, history } = this.props;
 
-        dispatch(submitAnswer(
-            lettersInSlots.length === WORD_LENGTH
-                ? this.joinLetters(lettersInSlots)
-                : '',
-        ));
+        if (isMobile()) {
+            dispatch(submitAnswer(
+                lettersInSlots.length === WORD_LENGTH
+                    ? this.joinLetters(lettersInSlots)
+                    : '',
+            ));
+        } else {
+            dispatch(submitAnswer(this.joinLetters(letters)));
+        }
```

Comment:
I should have thought about it earlier :|

Reply:
lots to do 🤯

Reply:
it's been two weeks since I wrote it but as far as I remember it's an important change of data flow / logic but it does not require a lot of changes in the code

Reply:
hmm, yeah... 🙃